go.mod

```diff
@@ -48,3 +49,7 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.2
 
 )
+
+replace github.com/projectcalico/felix => github.com/song-jiang/felix v0.0.0-20200808201846-cb49d31188dd
+
+replace github.com/projectcalico/cni-plugin => github.com/song-jiang/cni-plugin v1.11.1-0.20200809162107-4669c6f9bdc7
```

Comment:
Can we replace these with the real versions now if felix and cni-plugin have the Windows stuff merged?